src/python/bot/startup/run_cron.py

```diff
@@ -30,7 +30,9 @@
 
 CRON_TASKS = [
     'backup', 'sync_admins', 'batch_fuzzer_jobs', 'build_crash_stats',
-    'corpus_backup', 'fuzzer_coverage', 'predator_pull'
+    'corpus_backup', 'fuzzer_coverage', 'predator_pull',
```

Comment:
We should add a TODO to make this more list computed in a more automated way. e.g. dynamically importing isntead.